src/components/TipInput.vue

```diff
@@ -236,5 +252,19 @@ export default {
       }
     }
   }
+
+  .token-tips {
+    display: inline-block;
```

Comment:
I propose to move this to the Dropdown component because until opened it behaves like buttons that are inline-block by default.